bpm/src/main/java/org/jboss/pnc/bpm/BpmTask.java

```diff
@@ -150,6 +150,7 @@ public Long getProcessInstanceId() {
             throw new CoreException("Could not serialize process parameters '" +
                     parameters + "'.", e);
         }
+
         actualParameters.put("pncBaseUrl", config.getPncBaseUrl());
         actualParameters.put("taskId", taskId);
         return actualParameters;
```

Comment:
Maybe these two are not needed? i guess all such params are wrapped in processParameters.  
         actualParameters.put("pncBaseUrl", config.getPncBaseUrl());
         actualParameters.put("taskId", taskId);